source/GameInterface/Services/Clans/Patches/ClanNameChangePatch.cs

```diff
@@ -18,22 +14,29 @@ public class ClanNameChangePatch
     {
         private static ILogger Logger = LogManager.GetLogger<ClanNameChangePatch>();
 
-
         static bool Prefix(ref Clan __instance, TextObject name, TextObject informalName)
```

Comment:
Can this file be removed since we are using autosync?

Reply:
This cannot be removed as it's a call from client UI

Reply:
Unless autosync has a way to permit client changes